presto-spi/src/main/java/com/facebook/presto/spi/Subfield.java

```diff
@@ -262,6 +262,15 @@ public boolean isPrefix(Subfield other)
         return false;
     }
 
+    public Subfield tail(String name)
```

Comment:
`tail` sounds a bit weird. After googling for a while, I couldn't find a good name for it. Maybe @rongrong can help given she has been playing fully qualified name for a while?